src/stateless/FilterChip.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <div :class="{ 'filter-chip__active': active, 'filter-chip__open': isOpen }" class="filter-chip" @click="click()">
+        <div class="label">
+            <slot>{{ label }}</slot>
+        </div>
+        <div v-if="selectionCounter.all > 0" class="selection-counter">
+            {{ selectionCounter.selected }}/{{ selectionCounter.all }}
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    props: {
+        click: { type: Function, required: true },
```

Comment:
Instead of calling this function directly you can just emit an event: `@click="$emit('click')"`